src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+import duke.exception.IncompleteCommandException;
+import duke.exception.MissingIndexException;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String GREET_STRING = "Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?";
+    private static final String BYE_STRING = "Bye. Hope to see you again soon!";
+    private static final String PROMPT_GENERIC_INVALID_COMMAND = "I don't understand what you are saying. Please follow this syntax:";
+    private static final String PROMPT_ALL_FEATURES = "Recognised Commands:" + System.lineSeparator() +
+            "'list'     | show you all your tasks" + System.lineSeparator() +
+            "'mark'     | mark tasks by index once completed" + System.lineSeparator() +
+            "'unmark'   | unmark tasks by index to undo 'mark'" + System.lineSeparator() +
+            "'delete'   | delete tasks by index (irreversible!)" + System.lineSeparator() +
+            "'todo'     | create general task" + System.lineSeparator() +
+            "'deadline' | create task with deadline: include '/by' for deadline" + System.lineSeparator() +
+            "'event'    | create task as event: include '/at' for the date/time" + System.lineSeparator() +
+            "'bye'      | exit Duke";
+    private static final String PROMPT_CORRECT_DEADLINE = "example: `deadline Rush CS2113 Assignment /by today`";
+    private static final String PROMPT_CORRECT_EVENT = "example: `event Watch CS2113 Lecture /at 4-6pm on Friday`";
+    private static final String PROMPT_CORRECT_TODO = "example: `todo Make sure that the user knows I need some text here!`";
+    private static final String PROMPT_CORRECT_INDEX_USE = "when marking, unmarking or deleting, use syntax like: `delete 2` for task #2" + System.lineSeparator() +
+            "you can do so for any of the tasks in `list`";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        Scanner s = new Scanner(System.in);
```

Comment:
Maybe you could use a more descriptive name for the Scanner object. For example, scanner.